### PR TITLE
[SSA-2024] add program

### PR DESCRIPTION
**Pull request by somatorio**
Add initial program and speakers

### CODE REVIEW

**Review by don-code — CHANGES_REQUESTED** [review]
The images under `static/events` aren't needed - can you remove them? The ones under `assets/events` are the ones that should stay.

File: static/events/2024-salvador/speakers/rafael-levi-batista-costa.jpg
Comment (toshywoshy) [inline static/events/2024-salvador/speakers/rafael-levi-batista-costa.jpg:1]: Please remove as already requested

File: static/events/2024-salvador/speakers/lucas-ribeiro-mata.jpeg
Comment (toshywoshy) [inline static/events/2024-salvador/speakers/lucas-ribeiro-mata.jpeg:1]: Please remove as already requested

File: static/events/2024-salvador/speakers/lara-xavier.jpeg
Comment (toshywoshy) [inline static/events/2024-salvador/speakers/lara-xavier.jpeg:1]: Please remove as already requested

File: static/events/2024-salvador/speakers/giovana-do-nascimento-pena.png
Comment (toshywoshy) [inline static/events/2024-salvador/speakers/giovana-do-nascimento-pena.png:1]: Please remove as already requested

File: static/events/2024-salvador/speakers/everton-mendonca-de-jesus.jpeg
Comment (toshywoshy) [inline static/events/2024-salvador/speakers/everton-mendonca-de-jesus.jpeg:1]: Please remove as already requested

File: static/events/2024-salvador/speakers/eduardo-paim.png
Comment (toshywoshy) [inline static/events/2024-salvador/speakers/eduardo-paim.png:1]: Please remove as already requested

File: static/events/2024-salvador/speakers/clayton-cavaleiro.png
Comment (toshywoshy) [inline static/events/2024-salvador/speakers/clayton-cavaleiro.png:1]: Please remove as already requested

File: static/events/2024-salvador/speakers/camilla-martins.jpeg
Comment (toshywoshy) [inline static/events/2024-salvador/speakers/camilla-martins.jpeg:1]: Please remove as already requested

File: content/events/2024-salvador/speakers/waldemar.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
++++
+Title = "Waldemar Neto"
+Twitter = ""
+linkedin = ""
+image = "tbd.jpeg"
+type = "speaker"
+linktitle = "waldemar"
+
++++
```
Comment (toshywoshy) [inline content/events/2024-salvador/speakers/waldemar.md:9]: Speaker has no bio

File: content/events/2024-salvador/speakers/camila-viana.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
++++
+Title = "Camila Viana"
+Twitter = ""
+linkedin = ""
+image = "tbd.jpeg"
+type = "speaker"
+linktitle = "camila-viana"
+
++++
```
Comment (toshywoshy) [inline content/events/2024-salvador/speakers/camila-viana.md:9]: Speaker has no bio

File: content/events/2024-salvador/program/camilla-martins.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "IaC com Gemini"
+Type = "talk"
+Speakers = ["camilla-martins"]
++++
+
+
```
Comment (toshywoshy) [inline content/events/2024-salvador/program/camilla-martins.md:10]: Talk has no description

File: content/events/2024-salvador/program/clayton-cavaleiro.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "Minhas atuais buscas são em ter excelência em infraestrutura como código com terraform, onde tenho orgulho de conseguir ser referência nos ambientes onde trabalho, e por arquiteturas de sistemas distribuídos e tolerantes a falhas."
+Type = "talk"
+Speakers = ["clayton-cavaleiro"]
++++
+
+
```
Comment (toshywoshy) [inline content/events/2024-salvador/program/clayton-cavaleiro.md:10]: Talk has no description

File: content/events/2024-salvador/program/eduardo-paim.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "Reformar, sem quebrar: A Jornada de Migração da Plataforma"
+Type = "talk"
+Speakers = ["eduardo-paim"]
++++
+
+
```
Comment (toshywoshy) [inline content/events/2024-salvador/program/eduardo-paim.md:10]: Talk has no description

File: content/events/2024-salvador/program/everton-mendonça-de-jesus.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "Teste de Chaos no iFood - Explorando Litmus Chaos e Kubernetes"
+Type = "talk"
+Speakers = ["everton-mendonça-de-jesus"]
++++
+
+
```
Comment (toshywoshy) [inline content/events/2024-salvador/program/everton-mendonça-de-jesus.md:10]: Talk has no description

File: content/events/2024-salvador/program/lara-xavier.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "Monitoramento vs. Observabilidade: Transformando Dados em Ações Estratégicas"
+Type = "talk"
+Speakers = ["lara-xavier"]
++++
+
+
```
Comment (toshywoshy) [inline content/events/2024-salvador/program/lara-xavier.md:10]: Talk has no description

File: content/events/2024-salvador/program/lucas-ribeiro-mata.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "MLOps: do desenvolvimento ao monitoramento contínuo de modelos de IA"
+Type = "talk"
+Speakers = ["lucas-ribeiro-mata"]
++++
+
+
```
Comment (toshywoshy) [inline content/events/2024-salvador/program/lucas-ribeiro-mata.md:10]: Talk has no description

File: content/events/2024-salvador/program/rafael-levi-batista-costa.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "Projeto de Devops com BI para análise de desempenho de projetos de software"
+Type = "talk"
+Speakers = ["rafael-levi-batista-costa"]
++++
+
+
```
Comment (toshywoshy) [inline content/events/2024-salvador/program/rafael-levi-batista-costa.md:10]: Talk has no description

**Review by toshywoshy — CHANGES_REQUESTED** [review]

**Review by don-code — APPROVED** [review]

**Comment by don-code** [comment]
I think this is fine. It looks like @toshywoshy will also need to reapprove before it can be merged.

**Review by toshywoshy — APPROVED** [review]

**Pull request merged**